server/routes/api/user.ts

```diff
@@ -3,6 +3,8 @@ import * as path from "path";
 import * as express from "express";
 import * as json2csv from "json2csv";
 import * as archiver from "archiver";
+import {mongoose} from "../../common";
```

Comment:
Using `User.findById()` instead would mean that we wouldn't have to import mongoose here and wouldn't have to wrap IDs in `mongoose.Types.ObjectId()`